config/context.tsx

```diff
@@ -0,0 +1,37 @@
+import { createContext, useContext, useReducer, ReactNode, Dispatch } from 'react';
```

Comment:
Lets change the name of the file to represent what type of context it is storing.
Also we should move this file to a more appropiate folder. This file does not contain settings